song/db/migrations/20230322025430_create_users_table.sql.sql

```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+CREATE TABLE IF NOT EXISTS users (
```

Comment:
```suggestion
CREATE TABLE users (
```
`IF NOT EXISTS`는 사용할 필요가 없습니다. 저번에 같이 해결했던건데 또 문제가 생겼나요?